molecule/provisioner/ansible.py

```diff
@@ -407,6 +407,10 @@ def default_options(self):
 
     @property
     def default_env(self):
+        # Allow user to use ~/path/to/foo in environment variables
+        for key, value in self._config.env.items():
```

Comment:
This will fail ugly with variables that contain list of paths, like PATH itself. Expand is not able to deal with path lists.

Reply:
I guess you could do something like split on `:` and apply it to each item, like

```python
self._config.env[key] = ':'.join([os.path.expanduser(entry) for entry in value.split(':')])
```

It would cause an issue if there was an environment variable that contained the pattern `:~`, but wasn't representing a directory, though I'm not sure how likely that is.

Reply:
I find it risky and sooner or later someone would hate us is we do it. On the other hand, lets go for a safe approach where we expand only this variable.

Can you please update the review to do this?